app/assets/stylesheets/main.scss

```diff
@@ -32,8 +31,99 @@ body {
       font-size: 25px;
     }
   }
+
+  ul{
+    list-style:none;
+    display: inline-block;
+  }
+
+  ul li{
+    display: inline-block;
+    padding: 10px 20px;
+    cursor:pointer;
+    transition:0.3s;
+  }
+
+  ul li:hover{
+    background-color: rgba(193, 156, 192, 0.72);
+  }
+
+  ul li.focused{
+    color:#fff;
+    background-color: #783b79;
+  }
+
+  a.menu-btn {
+    font-size: 30px;
+    float: right;
+    padding: 10px;
+    padding-right: 20px;
+    cursor: pointer;
+  }
+}
+
+.menu {
+  display:block;
+
+  $menu-width:250px;
+
+  >div {
+    position:fixed;
+    z-index:2;
+    top:0;
+    width:$menu-width;
+    height:100%;
+    box-sizing: border-box;
+    -moz-box-sizing: border-box;
```

Comment:
We're using a tool called [Autoprefixr](https://github.com/postcss/autoprefixer) to add vendor-specific prefixes to all of our CSS rules. That means you can just use `box-sizing`, not `-moz-box-sizing`, etc.